### PR TITLE
Introduce experimental flag for cgroup

### PR DESCRIPTION
**Pull request by ruflin**
Make cgroup a config option

### CODE REVIEW

File: metricbeat/docs/fields.asciidoc [hunk 1]
```diff
@@ -3193,7 +3193,7 @@ The hard limit on the number of file descriptors opened by the process. The hard
 [float]
 == cgroup Fields
 
-Metrics and limits from the cgroup of which the task is a member. cgroup metrics are reported when the process has membership in a non-root cgroup. These metrics are only available on Linux.
+EXPERIMENTAL: Metrics and limits from the cgroup of which the task is a member. cgroup metrics are reported when the process has membership in a non-root cgroup. These metrics are only available on Linux.
```
Comment (ruflin) [inline metricbeat/docs/fields.asciidoc]: @dedemorton If I use warning, a nice box is generated. Do you know if something similar exists for EXPERIMENTAL?

Comment (dedemorton) [inline metricbeat/docs/fields.asciidoc]: @ruflin  You can use one of the following (see https://github.com/elastic/docs#experimental):
- experimental[] to get a warning box that says "This functionality is experimental and may be changed or removed completely in a future release."
- experimental[Custom text goes here] to add custom text. I'm not sure exactly how this one looks (never tried it), but give it a try!

Comment (ruflin) [inline metricbeat/docs/fields.asciidoc]: That is the way it looks now with `experimental[]` right now. The warning box shows up when hovering. I was hoping for some more warning colors, but that works.
screen shot 2016-08-18 at 08 17 04

File: metricbeat/module/system/process/cgroup.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
+package process
```
Comment (monicasarbu) [inline metricbeat/module/system/process/cgroup.go]: +1 for the idea of moving the code to cgroup.go.

**Comment by monicasarbu** [comment]
LGTM

**Comment by andrewkroh** [comment]
LGTM

**Pull request merged**